recipes/rsync/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rsync" %}
+{% set version = "3.1.2" %}
+{% set sha256 = "ecfa62a7fa3c4c18b9eccd8c16eaddee4bd308a76ea50b5c02a5840f09c0a1c2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.samba.org/ftp/{{ name }}/src/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: './configure --prefix=$PREFIX --without-included-zlib && make install'
```

Comment:
Is there a `make check`? Also any thoughts on breaking this out into `build.sh`?